my-fund-backend/src/main/java/com/myfund/model/DTO/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,73 @@
+package com.myfund.model.DTO.mapper;
+
+import com.myfund.model.Category;
+import com.myfund.model.DTO.CreateSubCategoryDTO;
+import com.myfund.model.DTO.CategoryDTO;
+import com.myfund.model.DTO.CreateCategoryDTO;
+import com.myfund.model.DTO.SubCategoryDTO;
+import com.myfund.model.SubCategory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CategoryMapper {
```

Comment:
for this mapper it will be good to have tests

Reply:
I have created tests for the CategoryMapper.